concepticondata/references/references.bib

```diff
@@ -1,3 +1,22 @@
+@book{SidwellAlves2021,
+  author       = {Sidwell, Paul and Alves, Mark},
+  title        = {Vietic 116 item phylogenetic lexicon [Data Set, Version 1.0]},
+  year         = {2021},
+  howpublished         = {Original Talk: The Vietic Languages: A Phylogenetic Analysis. Presented at the 9th International Conference on Austroasiatic Linguistics (ICAAL).},
```

Comment:
Did you verify that the talk was prsented on this meeting? I only verified the meeting for Sidwell2021 myself, but do not remember this talk was also shown there (or was it?) In any case: final `.` should be deleted behind `(ICAAL)`.

Reply:
Yes, it was presented on the first day of the meeting. I erased the final `.`.